### ISSUE TITLE
Change backbone

### ISSUE THREAD

**Issue by ntelo007** [oscarkey/multitask-learning#11]
Hi,

could you please help me change the backbone neural net (to be more specific with a ResNet34 pre-trained on ImageNet) and change the final tasks with 3 classification tasks?

Thank you for your time.

**Comment by versine** [comment]
This may be a complicated change due to the changes that had to be made to the base ResNet architecture in our implementation, and the complications of our training environment and tasks. 

If you would like to try anyways, here are the basic things you should look at:

We modified the torchvision ResNet101 implementation to include dilated convolutions and an ASPP module. The main implementation is in [encoder.py](https://github.com/oscarkey/multitask-learning/blob/master/multitask-learning/cityscapestask/encoder.py).

Since this is only the encoder portion of our network, we have to rename the torchvision resnet weights to match our architecture. See [model.py](https://github.com/oscarkey/multitask-learning/blob/c4503c044ca7a29bebd4e70e9e030524654e5d00/multitask-learning/cityscapestask/model.py#L57).

You would also want to change our loss criterion and our decoder to match your classification tasks. The most similar to a classifier in our version is the segmentation task, since it basically does classification per-pixel. See: [train.py](https://github.com/oscarkey/multitask-learning/blob/c4503c044ca7a29bebd4e70e9e030524654e5d00/multitask-learning/cityscapestask/train.py#L52) and
[decoders.py](https://github.com/oscarkey/multitask-learning/blob/c4503c044ca7a29bebd4e70e9e030524654e5d00/multitask-learning/cityscapestask/decoders.py#L1).

**Comment by ntelo007** [comment]
Could you please assist me on creating the loss function in Keras?
